### ISSUE TITLE
dbc.AccordionItem can only handle string child

### ISSUE THREAD

**Issue by JonThom** [facultyai/dash-bootstrap-components#760]
- dash version: `#2.0.0`
- dash-bootstrap-components version: `#1.0.0`
- components affected by bug: Accordion

### What is happening?

Passing a component as a child to dbc.AccordionItem results in the following exception:

> Objects are not valid as a React child (found: object with keys {namespace, props, type}). If you meant to render a collection of children, use an array instead.

### What should be happening?

The documentation includes examples where `AccordionItem` has components as children, so it should be expected to work.

### Code

```python
accordion = dbc.Accordion(
        [
            dbc.AccordionItem(
                [
                    html.P("This is the content of the first section"),
                    dbc.Button("Click here"),
                ],
                title="Item 1",
            ),
    ]
)
```

### Error messages

```
Objects are not valid as a React child (found: object with keys {namespace, props, type}). If you meant to render a collection of children, use an array instead.
    in div (created by AccordionBody)
    in div (created by t)
    in t (created by ForwardRef)
    in ForwardRef (created by ForwardRef)
    in ForwardRef (created by AccordionCollapse)
    in AccordionCollapse (created by AccordionBody)
    in AccordionBody (created by Ae)
    in div (created by AccordionItem)
    in AccordionItem (created by Ae)
    in div (created by Accordion)
    in Accordion (created by Ae)
    in Ae (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by Container)
    in Container (created by no)
    in no (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by Col)
    in Col (created by za)
    in za (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by Row)
    in Row (created by fb)
    in fb (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by Container)
    in Container (created by no)
    in no (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by Ib)
    in Ib (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by TabPane)
    in Ab (created by TabPane)
    in TabPane (created by sm)
    in div (created by TabContent)
    in TabContent (created by sm)
    in zb (created by TabContainer)
    in TabContainer (created by sm)
    in sm (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by ModalBody)
    in ModalBody (created by ud)
    in ud (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by ModalDialog)
    in div (created by ModalDialog)
    in ModalDialog (created by Modal)
    in div (created by Modal)
    in Modal (created by Modal)
    in Modal (created by od)
    in od (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by BaseTreeContainer)
    in div (created by Container)
    in Container (created by no)
    in no (created by CheckedComponent)
    in CheckedComponent (created by BaseTreeContainer)
    in ComponentErrorBoundary (created by BaseTreeContainer)
    in BaseTreeContainer (created by Context.Consumer)
    in Unknown (created by UnconnectedContainer)
    in div (created by UnconnectedGlobalErrorContainer)
    in div (created by GlobalErrorOverlay)
    in div (created by GlobalErrorOverlay)
    in GlobalErrorOverlay (created by DebugMenu)
    in div (created by DebugMenu)
    in DebugMenu (created by UnconnectedGlobalErrorContainer)
    in div (created by UnconnectedGlobalErrorContainer)
    in UnconnectedGlobalErrorContainer (created by withRadiumContexts(UnconnectedGlobalErrorContainer))
    in withRadiumContexts(UnconnectedGlobalErrorContainer) (created by Connect(withRadiumContexts(UnconnectedGlobalErrorContainer)))
    in Connect(withRadiumContexts(UnconnectedGlobalErrorContainer)) (created by UnconnectedContainer)
    in UnconnectedContainer (created by Connect(UnconnectedContainer))
    in Connect(UnconnectedContainer) (created by UnconnectedAppContainer)
    in UnconnectedAppContainer (created by Connect(UnconnectedAppContainer))
    in Connect(UnconnectedAppContainer) (created by AppProvider)
    in Provider (created by AppProvider)
    in AppProvider
```

**Comment by AnnMarieW** [comment]
Hi @JonThom 

This has been fixed recently - try upgrading to  1.0.1rc1

**Comment by JonThom** [comment]
Great! Thanks so Munch!
